be/src/exec/vectorized/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -0,0 +1,213 @@
+                    }
+                } else {
+                    nullable_column->append_nulls(1);
+                }
+            }
+            break;
+        }
+        case 4: {
+            // COMPLETE_TIME
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(4);
+                NullableColumn* nullable_column = down_cast<NullableColumn*>(column.get());
+                if (task_run_info.__isset.complete_time) {
+                    int64_t complete_time = task_run_info.complete_time;
+                    if (complete_time <= 0) {
+                        nullable_column->append_nulls(1);
+                    } else {
+                        DateTimeValue t;
+                        t.from_unixtime(complete_time, TimezoneUtils::default_time_zone);
+                        fill_column_with_slot<TYPE_DATETIME>(column.get(), (void*)&t);
+                    }
+                } else {
+                    nullable_column->append_nulls(1);
+                }
+            }
+            break;
+        }
+        case 5: {
+            // STATE
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(5);
+                const std::string* str = &task_run_info.state;
+                Slice value(str->c_str(), str->length());
+                fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&value);
+            }
+            break;
+        }
+        case 6: {
+            // DATABASE
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(6);
+                std::string db_name = SchemaHelper::extract_db_name(_db_result.dbs[_db_index - 1]);
+                Slice value(db_name.c_str(), db_name.length());
+                fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&value);
+            }
+            break;
+        }
+        case 7: {
+            // DEFINITION
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(7);
+                const std::string* str = &task_run_info.definition;
+                Slice value(str->c_str(), str->length());
+                fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&value);
+            }
+            break;
+        }
+        case 8: {
+            // ERROR_CODE
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(8);
+                if (task_run_info.__isset.error_code) {
+                    int64_t value = task_run_info.error_code;
+                    fill_column_with_slot<TYPE_BIGINT>(column.get(), (void*)&value);
+                } else {
+                    fill_data_column_with_null(column.get());
+                }
+            }
+            break;
+        }
+        case 9: {
+            // ERROR_MESSAGE
+            {
+                ColumnPtr column = (*chunk)->get_column_by_slot_id(9);
+                const std::string* str = &task_run_info.error_message;
```

Comment:
error_message is nullable?

Reply:
Yes, but there will be no NULL value when FE returns.